README.md

```diff
@@ -96,7 +96,18 @@ http://www.gnu.org/licenses/gpl.txt
 
 ## Installation
 
+### Flatpak
+
+The flatpak version is distributed on Flathub, and runs on most major Linux distributions. See instructions here: [Flathub application page](https://flathub.org/apps/details/io.github.antimicrox.antimicrox)
+
+If you have Flathub set up already:
```

Comment:
I would add hyperlink to :https://flatpak.org/setup/

If you have Flathub [set up ](https://flatpak.org/setup/) already.